svelte-kit-scss/src/lib/components/Profile/UserProfile/UserProfile.spec.ts

```diff
@@ -16,19 +16,55 @@ describe('UserProfile', () => {
     });
   });
 
-  it.each([
```

Comment:
I'm surprised to see you removed this in favour of the non-dynamic setup. What made you move in this direction?